roles/deploy/tasks/main.yml

```diff
@@ -82,5 +76,17 @@
   deploy_helper: path="{{ project_root }}" release="{{ deploy_helper.new_release }}" state=finalize
   when: project_finalize
 
-- name: Reload php-fpm
-  shell: sudo service php5-fpm reload
+- name: Run post_finalize_commands
+  shell: "{{ item }}"
+  args:
+    chdir: "{{ deploy_helper.new_release_path }}"
+  with_items: project_post_finalize_commands
+
+- name: Get current git commit
+  command: git rev-parse --short HEAD
+  args:
+    chdir: "{{ project_source_path }}"
+  register: git_commit
+
+- debug:
+    msg: "{{ project_version }}@{{ git_commit.stdout }} deployed as release {{ deploy_helper.new_release }}"
```

Comment:
Instead of adding a new task to "Get current git commit", try just adding `register: git_commit` to the existing "Clone project files" task. Then get the SHA with something like `{{ git_commit.after | truncate(7, true, '') }}`.